src/NzbDrone.Host/Startup.cs

```diff
@@ -158,6 +158,8 @@ public void ConfigureServices(IServiceCollection services)
                 {
                     { apikeyQuery, Array.Empty<string>() }
                 });
+
+                c.DescribeAllParametersInCamelCase();
```

Comment:
This gives camelCase names for properties on `PagingRequestResource` and other models